docs/user-guide/advanced/replicate_evaluations.md

```diff
@@ -140,6 +140,30 @@ IN_FEATURES=1024 \
 eva predict_fit --config configs/vision/pathology/offline/<task>.yaml
 ```
 
+### hibou-B (hist.ai) - DINOv2 ViT-B14 (1M Slides) [[7]](#references)
+To evaluate [hist.ai's](https://www.hist.ai/) FM with DINOv2 ViT-B14 backbone, pretrained on
+a proprietary dataset of one million slides, run: 
+
+```
+MODEL_NAME=pathology/histai_hibou_b \
+NORMALIZE_MEAN=[0.7068,0.5755,0.722] \
+NORMALIZE_STD=[0.195,0.2316,0.1816] \
+IN_FEATURES=768 \
+eva predict_fit --config configs/vision/pathology/offline/<task>.yaml
+```
+
+### hibou-L (hist.ai) - DINOv2 ViT-L14 (1M Slides) [[7]](#references)
+To evaluate [hist.ai's](https://www.hist.ai/) FM with DINOv2 ViT-L14 backbone, pretrained on
+a proprietary dataset of one million slides, run: 
```

Comment:
mention that histai needs to be contacted for download and link to https://www.hist.ai/#models

Reply:
I think the website is not up to date, the model is available on huggingface, same as `hubou-B`
https://huggingface.co/histai/hibou-L